src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Shapes/Shapes_Tests.cs

```diff
@@ -46,6 +46,34 @@ void TabWaitAndThenScreenshot(string buttonName)
 			}
 		}
 
+		[Test]
+		[AutoRetry]
+		public void Affect_Measurement_polygon()
+		{
+			Run("SamplesApp.Windows_UI_Xaml_Shapes.PolygonPage");
+
+			_app.WaitForElement("DPolygon");
+			_app.Marked("ClearShape").Tap();
+			TakeScreenshot($"PolygonPage - ClearShape");
+
+			_app.Marked("ChangeShape").Tap();
+			TakeScreenshot($"PolygonPage - ChangeShape-After clear");
```

Comment:
I'm wondering, can you look at the measure of the element ? Would also add some additional validation of the content of the shape.